packages/content/test/attributes/EmailAddress.test.ts

```diff
@@ -0,0 +1,59 @@
+import { ParsingError } from "@js-soft/ts-serval";
+import { EMailAddress } from "../../src";
+
+describe("creation of RepositoryAttributes of Attribute Value Type EMailAddress", function () {
```

Comment:
From now on we would like to establish the spelling "Attribute value type" instead of "Attribute Value Type". It would be great if you could adjust this everywhere in the test file for the EMailAddress and for the Website.